sdk/python/touca/cli/_post.py

```diff
@@ -94,8 +104,6 @@ def run(self):
         for batch_dir in src_dir.glob("*"):
             if not batch_dir.is_dir():
                 continue
-            if not batch_dir.name.endswith("-merged"):
-                continue
             batchNames.append(batch_dir.name[:-7])
```

Comment:
Should we continue to use `batch_dir.name[:-7]` (cut the last 7 characters) now that we are not checking `.endswith("-merged")`? Feels to me that we need:
```suggestion
            batchNames.append(
                batch_dir.name[:-7]
                if batch_dir.name.endswith("-merged")
                else batch_dir.name
            )
```